src/slim-select/render.ts

```diff
@@ -481,6 +490,18 @@ export default class Render {
         }
       }
     }
+
+    
+
+    // If the number of selected items goes above the threshold, hide the chips
+    if (selectedOptions.length > this.settings.selectedChipsLimit) {
+      this.main.values.classList.add(this.classes.valueChipsHidden)
```

Comment:
So we wouldnt add the class to values. the newly created div for max values would just replace the contents of values by setting innerhtml